src/Javis.jl

```diff
@@ -21,8 +21,21 @@ end
     Video(width, height)
 
 Create a video with a certain `width` and `height` in pixel.
+This also sets `CURRENT_VIDEO`.
```

Comment:
I think it would be wise to explain what this variable's purpose is and why we are using it here.

Reply:
Done. Good point!